api/payloads/manifest.go

```diff
@@ -264,7 +264,7 @@ func (s ManifestApplicationService) Validate() error {
 	return validation.ValidateStruct(&s, validation.Field(&s.Name, validation.Required))
 }
 
-var unitAmount = regexp.MustCompile(`^\d+(?:B|K|KB|M|MB|G|GB|T|TB)$`)
```

Comment:
Have you verified what is the effect of accepting lower case letters?

If Korifi translates those units directly to k8s, then this might have unwanted behaviour, for example:

```
Pay attention to the case of the suffixes. If you request 400m of memory, this is a request for 0.4 bytes. Someone who types that probably meant to ask for 400 mebibytes (400Mi) or 400 megabytes (400M).

```
https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory

Reply:
forget my previous comment - obviously i'm getting mebibytes... i'll have another look

Reply:
Maybe a stupid question, but shouldn't we also accept `b`, `k` and `kb`? The v3 docs state that all units can be in upper or lower case.

Reply:
Good point - i will also add b, k and kb.

I found that we hardcode our request and limits to mebibyte... is this on purpose? to cleanly fix this, i think, i would also adapt them to reflect actual MegaByte. What do you think? Is the cfprocess_controller.go the right place to implement this?

https://github.com/cloudfoundry/korifi/blob/f60426c07faa7bd37bbff1bc8794eae21c276cf5/controllers/controllers/workloads/cfprocess_controller.go#L314

Reply:
I have no memory why we have preferred mebibytes. I have posted a [question](https://cloudfoundry.slack.com/archives/C0297673ASK/p1714044427954889) on slack in case anyone remembers

Reply:
Here is the issue that describes that decision. Apparently docs are wrong :)
https://github.com/cloudfoundry/korifi/issues/720

Reply:
I will approve this PR in order not to age. If anyone needs the other units (`b`, `k`, `kb`) we could introduce them later.